Example/BonMot/BONSpecialGenerator.swift

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env swift
```

Comment:
It looks like you've duplicated the generator script. If you're going to reference it in the project (which is not a bad idea), it should either a) be put in as a file reference, or b) moved on the file system. Also, this gave me the idea for https://github.com/Raizlabs/BonMot/issues/89